code/bit_manipulation/src/clear_bits_from_msb/clear_bits_from_msb.py

```diff
@@ -0,0 +1,23 @@
+'''
+Problem Statement :
+Clear all MSB's except for the first i LSB's of a number n.
+MSB -> Most Significant Bit
+LSB -> Most Significant Bit 
+'''
+
+def clearBitsFromMsb(n, i):
```

Comment:
For Python keep lowercamel case. Also reformat this file using black.

Reply:
Done.